x/metadata/client/cli/tx.go

```diff
@@ -1264,19 +1302,31 @@ func validateAccAddress(addr, argName string) (string, error) {
 	return addr, nil
 }
 
-// ParseNetAssetValueString splits string (example 1hotdog,2jackthecat) to list of NetAssetValue's
+// ParseNetAssetValueString splits string (example 1hotdog,1;2jackthecat100,...) to list of NetAssetValue's
 func ParseNetAssetValueString(netAssetValuesString string) ([]types.NetAssetValue, error) {
-	navs := strings.Split(netAssetValuesString, ",")
+	navs := strings.Split(netAssetValuesString, ";")
 	if len(navs) == 1 && len(navs[0]) == 0 {
 		return []types.NetAssetValue{}, nil
 	}
 	netAssetValues := make([]types.NetAssetValue, len(navs))
 	for i, nav := range navs {
-		coin, err := sdk.ParseCoinNormalized(nav)
+		parts := strings.Split(nav, ",")
```

Comment:
It'd probably be more user friendly to use the format `[<num>]<denom>[:<volume>]`. Both the `<num>` and `<volume>` would both default to `1`. Each entry would then be comma separated. So the example would look like `1hotdog:1,2jackthecat:100`. The `:` is often used to indicate a link between the left and right side, so it's natural to use there. And then the commas separate each entry similar to a `Coins` string, which make that a fairly natural fit too.

Reply:
Nevermind. I just saw that the `x/marker` version of this command also uses the `<num><denom>,<volume>` format with `;` separators. It's probably best to be consistent, so let's leave it as is and maybe change it some other day if we think of it.